src/index.js

```diff
@@ -291,37 +289,11 @@ export class ULabel {
             }
         })
 
+        // This listener does not use jquery because it requires being able to prevent default
+        // There are maybe some hacky ways to do this with jquery
+        // https://stackoverflow.com/questions/60357083/does-not-use-passive-listeners-to-improve-scrolling-performance-lighthouse-repo
         // Detection ctrl+scroll
-        document.getElementById(ul.config["annbox_id"]).addEventListener("wheel", function (wheel_event) {
-            // Prevent scroll-zoom
-            wheel_event.preventDefault();
-            let fms = ul.config["image_data"].frames.length > 1;
-            if (wheel_event.altKey) {
-                // When in brush mode, change the brush size
-                if (ul.subtasks[ul.state["current_subtask"]]["state"]["is_in_brush_mode"]) {
-                    ul.change_brush_size(wheel_event.deltaY < 0 ? 1.1 : 1 / 1.1);
-                }
-            } else if (fms && (wheel_event.ctrlKey || wheel_event.shiftKey || wheel_event.metaKey)) {
-                // Get direction of wheel
-                const dlta = Math.sign(wheel_event.deltaY);
-                ul.update_frame(dlta);
-            } else {
-                // Don't scroll if id dialog is visible
-                if (ul.subtasks[ul.state["current_subtask"]]["state"]["idd_visible"] && !ul.subtasks[ul.state["current_subtask"]]["state"]["idd_thumbnail"]) {
-                    return;
-                }
-
-                // Get direction of wheel
-                const dlta = Math.sign(wheel_event.deltaY);
-
-                // Apply new zoom
-                ul.state["zoom_val"] *= (1 - dlta / 5);
-                ul.rezoom(wheel_event.clientX, wheel_event.clientY);
-
-                // Only try to update the overlay if it exists
-                ul.filter_distance_overlay?.draw_overlay()
-            }
-        }, {"signal": signal});
+        document.getElementById(ul.config["annbox_id"]).addEventListener("wheel", ul.handle_wheel.bind(ul));
```

Comment:
Does this not require the `signal`?

Reply:
doesnt seem to need it to work

Reply:
If it still detaches correctly with `remove_listeners` then seems fine to me. Maybe anything attached to the `annbox` doesn't need detaching.

Speaking of which, is [this line](https://github.com/SenteraLLC/ulabel/blob/657f7bde825be664822d7b1e4b08cb825caa1f63/src/index.js#L145) in `remove_listeners` correct? It looks like it selects the `annbox` but doesn't call `.off(".ulabel")` on it.

Additionally, is the `AbortController` used at all now? Can probably remove it if not.

Reply:
good catch, ill that do-nothing line.

yeah in my testing using a react app, the `remove_listeners` now actually removes every ulabel listener. granted this is happening at the same time that the html is all destroyed, so i presume that if you were to call `remove_listener` while the ulabel html was all still on the page, it wouldn't remove this one specifically.

not sure when you would ever do that since without listeners ulabel doesnt work anyways, but maybe for completeness i should add that back? idk

Reply:
alternatively i can call `removeEventListener` with the same function input and it should remove it

Reply:
actually, looks like currently there are several listeners that are not removed by `remove_listeners` because they are attached to specific elements. Rather than explicitly remove them all, i think i'll note that the `remove_listeners` function is designed to remove `document` and `window` listeners specifically, since all others will be garbage collected along with their html elements

Reply:
[54eb01a](https://github.com/SenteraLLC/ulabel/pull/160/commits/54eb01a3ae641e442bc976091826e9786ade7fa4)